tensilelite/Tensile/KernelWriter.py

```diff
@@ -1488,7 +1487,9 @@ def graIncrementsAB():
       gsuBackup   = kernel["GlobalSplitU"]
       gsuLabel    = Label(label=self.labels.getNameInc("GSU"), comment="")
       gsuLabelEnd = Label(label=self.labels.getNameInc("GSU_End"), comment="")
-      module.add(SCmpEQU32(src0=sgpr("GSU"), src1=1, comment="GSU == 1 ?"))
+      with self.allocTmpSgpr(1) as tmpSgprGSU:
+        module.add(SAndB32(dst=sgpr(tmpSgprGSU.idx), src0=sgpr("GSU"), src1=hex(0x3FFF), comment="Restore GSU"))
+        module.add(SCmpEQU32(src0=sgpr(tmpSgprGSU.idx), src1=1, comment="GSU == 1 ?"))
```

Comment:
Can we use hex(1) with SAndB32 to change the state of SCC?

Reply:
This can't be done with SAndB32 because SCC=(D!=0) is set in SAndB32. It only compare zero and non-zero cases instead of "==1".